h2o-automl/src/main/java/ai/h2o/automl/leaderboard/LeaderboardColumn.java

```diff
@@ -0,0 +1,16 @@
+package ai.h2o.automl.leaderboard;
+
+import hex.Model;
+import water.Freezable;
+import water.Key;
+
+public interface LeaderboardColumn<V, SELF extends LeaderboardColumn> extends Freezable<SELF> {
+    LeaderboardColumnDescriptor getDescriptor();
+
+    Key<Model> getModelId();
```

Comment:
@seb-h2o not used

Reply:
I'll still leave it for consistency and debuggability: leaderboard stores the columns in a map, that's why it's not used, but when I have a column, I want to be able to know to which model it is associated.
Also, if I decided to store them in a list, the modelId would be necessary: the interface should be self-contained and independent from the data structure I'll use to store the instances.

Reply:
To say differently, a column entry without anything to identify its row index can quickly become unusable.

Reply:
I may rename the interface to `LeaderboardCell`, and `LeaderboardColumnDescriptor` to `LeaderboardColumn`.
This would make more sense indeed.

Reply:
@deil87 : client mode proved me that it was necessary.
The `IcedHashMap<Key, LeaderboardCell[]>` raised `ClassCastException` after deserialization in client mode, so I changed the storage as a simple `LeaderboardCell[]` and it now works :)